clients/csharp/resnet/Program.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using CSharpClient;
+using Grpc.Core;
+using Newtonsoft.Json;
+using Tensorflow.Serving;
+
+namespace resnet
+{
+    internal class Program
+    {
+        private static readonly string errorMessage = "" + Environment.NewLine + "dotnet resnet.dll [url(without port)] [path/to/local/image] <use_ssl (true/false - false if absent)> <auth_key (use_ssl must be true)>";
+
+        private static int Main(string[] args)
+        {
+            return MainAsync(args).Result;
+        }
+
+        private static async Task<int> MainAsync(string[] args)
+        {
+            if (args.Length < 2)
+            {
+                Console.WriteLine($"Use as {errorMessage}");
+                return 1;
+            }
+
+            var host = args[0];
+            var image = args[1];
+
+            var useSSL = false;
+
+            if (args.Length > 2)
+            {
+                var useSslString = args[2];
+                var parsed = bool.TryParse(useSslString, out useSSL);
+                Console.WriteLine(useSSL ? "Using SSL" : "Not using SSL");
+            }
+
+            string auth = null;
+            if (args.Length > 3 && useSSL)
+            {
+                auth = args[3];
+                Console.WriteLine(!string.IsNullOrEmpty(auth) ? "Using auth" : "Not using auth");
+            }
+
+            var client = new ScoringClient(host, useSSL ? 443 : 80, useSSL, auth, "my-aks-service");
+            //var client = new ScoringClient(host, 50051, useSSL, auth);
+
+            using (var content = File.OpenRead(image))
+            {
+                IScoringRequest request = new ImageRequest("Placeholder:0", content);
+                var result = await client.ScoreAsync<float[,]>(request);
```

Comment:
Even though this sample is noted as for resnet50, it might be useful to note the fact that ssd-vgg will send back an array of rank 5 so this line would be 
```
var result = await client.ScoreAsync<float[,,,,]>(request);
```